docs/tutorials/start-a-private-network/4customchain.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Creating Your Private Network
+---
+Now that each participant has their own keys generated, you're ready to start your own custom chain.
+
+> Validators should not share the same keys, even for learning purposes. If two validators have the same keys, they will produce conflicting blocks, and be slashed.
+
+## Create a Chainspec
```

Comment:
```suggestion
## Create a Chain Specification
```

I think it is "chain spec" rather than one word. Would recommend doing a find replace here.